layouts/partials/BasicAgentUsage-nix.html

```diff
@@ -76,14 +76,13 @@
 
 More information about the metrics, events and service checks for an integration can be retrieved with the check command:
 
-    <%= $sudo %><%= @item[:servicename] %> check [integration]
+<%= $sudo %><%= @item[:servicename] %> check &#91;integration&#93;
 
 Add the check_rate argument to get the most recent values for rates:
 
-    <%= $sudo %><%= @item[:servicename] %> check [integration] check_rate
+<%= $sudo %><%= @item[:servicename] %> check &#91;integration&#93; check_rate
 
-<%= $sudo %>
-  <%= @item[:servicename] %> check [integration]
+<%= $sudo %><%= @item[:servicename] %> check &#91;integration&#93;
```

Comment:
you pointed out the real issue. it wasn't the brackets. well it is when its not code. being indented 2 tabs (4 spaces) means the markdown is not interpreted and nanoc doesn't complain. this extra line was not indented. my fix fixed it, but the fix worked around the real issue.